src/views/Practice/Practice.jsx

```
@@ -77,6 +77,7 @@ const Practice = () => {
     const audio = new Audio(levelCompleteAudioSrc);
     audio.play();
     callConfetti();
+    window.telemetry?.syncEvents && window.telemetry.syncEvents();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)